### ISSUE TITLE
Latest commit broke theme demo on Hugo Themes website

### ISSUE THREAD

**Issue by onedrawingperday** [spookey/slick#8]
Hello @spookey 

The Goldmark configuration settings you have [introduced](https://github.com/spookey/slick/commit/d6a08bc23989d42cbac601989995395d52c0b695) result in the following ERROR:
```
2:12:29 AM: * 'Goldmark' expected a map, got 'slice'
2:12:29 AM: * 'Highlight' expected a map, got 'slice'
2:12:29 AM: FAILED to create exampleSite for slick
```

Please have a look at the sample configuration settings in [Hugo Docs](https://gohugo.io/getting-started/configuration-markup#readout) and amend the exampleSite config.

**Comment by spookey** [comment]
Hey @onedrawingperday 
thanks for the hint

I tried to fix it in 14fdc3d

Hopefully it works now..

**Comment by onedrawingperday** [comment]
@spookey 

Thanks for addressing this issue. I confirm the fix and I have updated your theme in https://github.com/gohugoio/hugoThemes/commit/fa0f586134b52e26b8f5beb3371fd9962ab2fbac
